message_definitions/v1.0/planckAero.xml

```diff
@@ -73,9 +73,15 @@
             <entry value="4" name="PLANCK_DECK_SPOOL_EOL_WARNING"/>
             <entry value="5" name="PLANCK_DECK_SPOOL_EOL_ERROR"/>
         </enum>
-	<enum name="PLANCK_CMD_REQ_TYPE"> <!-- Extending with DECK LID commands -->
-            <entry value="5" name="PLANCK_CMD_REQ_DECK_LID_OPEN"/>
-            <entry value="6" name="PLANCK_CMD_REQ_DECK_LID_CLOSE"/>
+        <enum name="PLANCK_CMD_REQ_TYPE"> <!-- Extending with DECK LID commands -->
+            <entry value="5" name="PLANCK_CMD_REQ_DECK_LID_OPEN">
+                <description>Enable Planck stateinfo data</description>
+                <param index="1" label="Force" minValue="0" maxValue="21196" increment="21196">21196: force open (override checks)</param>
```

Comment:
Should we expand on the label to 
> 0: open unless prevented by safety checks, 21196: force open (override checks)

Reply:
I had in mind that any value != 21196: run the checks otherwise was have a cmd that doesn't do anything for all the values in between. But we can change if you prefer

Reply:
I see. I like your suggestion. I was really just looking at the example provided in APM, where they call out "0" as normal behavior vs using a forcing key value. See below. 

https://ardupilot.org/dev/docs/mavlink-arming-and-disarming.html